build.gradle

```diff
@@ -109,8 +109,8 @@ allprojects {
         }
         json {
             target "src/**/*.json"
-            // schemaVersion must be at the top or else Fabric will complain
```

Comment:
Why are you removing this exclude?

Reply:
Sorry, I could not get multiple excludes to work initially so this one was moved to the fabric build.gradle but I fixed the issue and moved it back.